spec/requests/api/blueprints_spec.rb

```diff
@@ -41,10 +41,13 @@
 
     it "will show a blueprint's bundle if requested" do
       blueprint = FactoryGirl.create(:blueprint, :name => "Test Blueprint")
-      service_catalog = FactoryGirl.create(:service_template_catalog)
```

Comment:
Oh, I see, they've just been moved. I think if you move them back they won't be included in the diff unnecessarily